### ISSUE TITLE
Dropdown menu in mobile view not updating aria-expanded state

### ISSUE THREAD

**Issue by kensgists** [paypal/bootstrap-accessibility-plugin#14]
The state on the button remains aria-expanded="false" regardless of whether the menu is expanded or collapsed. (Should have caught this last time I checked the issue. Sorry guys. Thanks for the work on this. Already being used in projects at my university!)
